src/main/java/gr/cytech/chatreminderbot/rest/controlCases/CaseSetConfigurations.java

```diff
@@ -0,0 +1,52 @@
+package gr.cytech.chatreminderbot.rest.controlCases;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.transaction.Transactional;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CaseSetConfigurations {
+    @PersistenceContext(name = "wa")
+    public EntityManager entityManager;
+
+    private ArrayList<String> splitMsg;
+
+    public void setSplitMsg(ArrayList<String> splitMsg) {
```

Comment:
wouldn't it be much better if you passed this as a method parameter, rather than having it as an instance field that *must* be set prior to doing anything else with the instance?